File: JS-Basics/Conditional Statements/Exercise/godzillaVsKong.js
function godzillaVsKong(input) {
    let budget = Number(input[0]);
    let extras = Number(input[1]);
    let clothingPrice = Number(input[2]);

    let decorationPrice = budget * 0.1;

    if (extras > 150) {
        clothingPrice *= 0.9;
    }

    let totalPrice = extras * clothingPrice + decorationPrice;

    if (totalPrice > budget) {
        let remainingMoney = totalPrice - budget;
        console.log('Not enough money!');
        console.log(`Wingard needs ${remainingMoney.toFixed(2)} leva more.`);
    } else {
        remainingMoney = budget - totalPrice;
        console.log('Action!');
        console.log(`Wingard starts filming with ${remainingMoney.toFixed(2)} leva left.`);
    }
}

godzillaVsKong(["20000","120","55.5"]);

File: Python/SoftUni PB/Exercises/Python Basics/Conditional Statements - Exercise/Godzilla_vs_Kong.py
budget = float(input())
statists = int(input())
clothing = float(input())

decor = budget * 0.1
clothing_price = statists * clothing

if statists > 150:
    clothing_price *= 0.9

needed_money = decor + clothing_price
difference = abs(needed_money - budget)

if needed_money > budget:
    print("Not enough money!")
    print(f"Wingard needs {difference:.2f} leva more.")
else:
    print("Action!")
    print(f"Wingard starts filming with {difference:.2f} leva left.")
